sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_critical_result_inference_async.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import asyncio
+import datetime
+import os
+
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights.aio import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
+
+"""
+FILE: sample_critical_result_inference_async.py
+
+DESCRIPTION:
+The sample_critical_result_inference_async.py module processes a sample radiology document with the Radiology Insights service.
+It will initialize an asynchronous RadiologyInsightsClient, build a Radiology Insights request with the sample document,
+submit it to the client, RadiologyInsightsClient, build a Radiology Insights job request with the sample document,
+submit it to the client and display the Critical Results description extracted by the Radiology Insights service.     
+
+
+USAGE:
+   
+"""
+
+
+class HealthInsightsSamples:
+    async def radiology_insights_async(self) -> None:
+        KEY = os.environ["AZURE_HEALTH_INSIGHTS_API_KEY"]
+        ENDPOINT = os.environ["AZURE_HEALTH_INSIGHTS_ENDPOINT"]
+
+        # Create a Radiology Insights client
+        # <client>
```

Comment:
Nit: doesnt seem like this line is useful

Reply:
agree, removed and added snippet functionality